plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_it.properties

```diff
@@ -4,72 +4,80 @@
+command.org.jkiss.dbeaver.core.navigator.bookmark.add.name             = Aggiungi Segnalibro
+command.org.jkiss.dbeaver.core.navigator.bookmark.navigate.description = Naviga all'oggetto del segnalibro
+command.org.jkiss.dbeaver.core.navigator.bookmark.navigate.name        = Naviga verso
+command.org.jkiss.dbeaver.core.new.connection.description              = Wizard Nuova Connessione
+command.org.jkiss.dbeaver.core.new.connection.from.url.description     = Crea una nuova connessione a partire dalla URL JDBC
+command.org.jkiss.dbeaver.core.new.connection.from.url.name            = Nuova connessione da URL JDBC
+command.org.jkiss.dbeaver.core.new.connection.from.url.name_short      = Da URL JDBC ...
+command.org.jkiss.dbeaver.core.new.connection.name                     = Nuova Connessione
+command.org.jkiss.dbeaver.core.process.stop.description                = Termina processo in esecuzione
+command.org.jkiss.dbeaver.core.process.stop.name                       = Termina processo
+command.org.jkiss.dbeaver.core.qm.clear.description                    = Pulisci il log del gestore query
+command.org.jkiss.dbeaver.core.qm.clear.name                           = Pulisci il log
+command.org.jkiss.dbeaver.core.qm.filter.description                   = Imposta filtro gestore query
+command.org.jkiss.dbeaver.core.qm.filter.name                          = Filtri ...
+command.org.jkiss.dbeaver.core.qm.refresh.description                  = Aggiorna storico delle query
+command.org.jkiss.dbeaver.core.qm.refresh.name                         = Aggiorna storico delle query
+command.org.jkiss.dbeaver.core.rollback.description                    = Rollback delle modifiche nella Sessione SQL
+command.org.jkiss.dbeaver.core.rollback.name                           = Rollback
+command.org.jkiss.dbeaver.core.rollbacklabel                           = Rollback
+command.org.jkiss.dbeaver.core.transaction_mode.name                   = Modalit\u00E0 di transazione
+command.org.jkiss.dbeaver.core.txn.autocommit.description              = Modifica la modalit\u00E0 transazionale della sessione attiva
+command.org.jkiss.dbeaver.core.txn.autocommit.name                     = Auto-commit
+command.org.jkiss.dbeaver.core.txn.log.description                     = Log delle transazioni correnti (query eseguite)
+command.org.jkiss.dbeaver.core.txn.log.name                            = Log delle transazioni
+command.org.jkiss.dbeaver.core.txn.pending.description                 = Elenco di tutte le transazioni appese
+command.org.jkiss.dbeaver.core.txn.pending.name                        = Transazioni appese
+command.org.jkiss.dbeaver.ui.tools.menu.description                    = Mostra il menu contestuale con gli strumenti del database
+command.org.jkiss.dbeaver.ui.tools.menu.name                           = Menu contestuali
 
 content-type.org.jkiss.dbeaver.bookmark.name = Segnalibro
 
+keyword.org.jkiss.dbeaver.core.keyword.pref.editor.erd.label = Drigramma ERD
```

Comment:
Diagramma maybe?

Reply:
Thank you @LonwoLonwo for your review, I have fixed this typo.